### ISSUE TITLE
Volume restart from backend with MDS failback fails

### ISSUE THREAD

**Issue by redlicha** [openvstorage/volumedriver#33]
Version info:

revision: 0b5a92a0afc4abeed9c7475411142da7d0a1840e
build time: Sun May 22 23:04:22 UTC 2016

```
2016-05-31 15:10:14:568741 +0200 CEST -- notice -- VolManager -- Backend Restart, Namespace: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c, owner tag 35, START -- [0x00007f17717fa700]
[...]

2016-05-31 15:10:14:750168 +0200 CEST -- error -- MDSMetaDataStore -- MDSMetaDataStore: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c: failed to connect to mds://172.19.10.41:26300: No route to host -- [0x00007f17717fa700]
2016-05-31 15:10:14:750195 +0200 CEST -- info -- MDSMetaDataStore -- build_new_one_: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c: attempting failover to mds://172.19.10.40:26300 -- [0x00007f17717fa700]
2016-05-31 15:10:14:750215 +0200 CEST -- info -- MDSMetaDataStore -- connect_: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c: connecting to mds://172.19.10.40:26300 -- [0x00007f17717fa700]
2016-05-31 15:10:14:750241 +0200 CEST -- info -- MDSMetaDataBackendHelpers -- make_db: mds://172.19.10.40:26300: running in-process, using fast path -- [0x00007f17717fa700]
2016-05-31 15:10:14:750268 +0200 CEST -- info -- MDSMetaDataBackend -- MDSMetaDataBackend: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c: using mds://172.19.10.40:26300 -- [0x00007f17717fa700]
2016-05-31 15:10:14:750398 +0200 CEST -- info -- MDSMetaDataBackend -- init_: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c: used clusters: 16384 -- [0x00007f17717fa700]
2016-05-31 15:10:14:784418 +0200 CEST -- info -- MDSMetaDataBackend -- lastCorkUUID: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c:  d8be60d7-822e-42d1-b124-2fa8671d9cc2 -- [0x00007f17717fa700]
2016-05-31 15:10:14:784563 +0200 CEST -- info -- MDSMetaDataBackend -- scrub_id: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c: scrub ID  e9e363c8-c040-4703-a56f-7eeb6db32c7b -- [0x00007f17717fa700]
2016-05-31 15:10:14:785141 +0200 CEST -- info -- CachedMetaDataStore -- init_pages_: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c: page capacity (entries): 256, max cached pages: 8192 -- [0x00007f17717fa700]
2016-05-31 15:10:14:785190 +0200 CEST -- info -- MetaDataServerTable -- set_role: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c: transition from MASTER to MASTER requested -- [0x00007f17717fa700]
2016-05-31 15:10:14:785292 +0200 CEST -- info -- BackendRestartAccumulator -- BackendRestartAccumulator: cork range: ( d8be60d7-822e-42d1-b124-2fa8671d9cc2, --] -- [0x00007f17717fa700]

[...]
 
2016-05-31 15:10:14:797357 +0200 CEST -- notice -- VolManager -- Backend Restart, Namespace: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c, owner tag 35, EXCEPTION -- [0x00007f17717fa700]
2016-05-31 15:10:14:797428 +0200 CEST -- error -- VFSLocalNode -- restart_volume_from_backend_: 5aa40950-37cc-4b18-bf9c-23f4c8779c5c: failed to restart volume: : start cork not seen d8be60d7-822e-42d1-b124-2fa8671d9cc2 -- [0x00007f17717fa700]
```

**Comment by redlicha** [comment]
Issue is fixed by PR #11 and specifically 29f97a0a9c3b0ff2c84726e21f1424fda66cb887 which was merged into the dev branch a day after the above dev build was created.

**Comment by redlicha** [comment]
New dev packages including the fix were uploaded to unstable.
